.github/workflows/release.yml

```diff
@@ -0,0 +1,90 @@
+name: Release
+on:
+  push:
+    tags:
+      - "v*"
+jobs:
+  ci:
+    # A branch is required, and cannot be dynamic - https://github.com/actions/runner/issues/1493
+    uses: ./.github/workflows/ci.yml
+  e2e: 
+    uses: ./.github/workflows/e2e-tests.yml
+  javy-plugin:
+    uses: ./.github/workflows/javy-plugin.yml
```

Comment:
Is this really needed? I think it's enough to have e2e and CI to pass, the javy-plugin is going to be handled by them